packages/types/src/options.ts

```diff
@@ -229,6 +229,8 @@ export interface ClientOptions<TO extends BaseTransportOptions = BaseTransportOp
    * Note that you must return a valid event from this callback. If you do not wish to modify the event, simply return
    * it at the end. Returning `null` will cause the event to be dropped.
    *
+   * TODO v8: Narrow the response type to `ErrorEvent` - this is technically a breaking change.
+   *
```

Comment:
Maybe let's add this as a normal comment instead of JS doc because we probably don't want this to show up for users.